packages/blockchain-link-utils/src/solana.ts

```diff
@@ -169,6 +218,46 @@ export function getAmount(
     return accountEffect.amount.toString();
 }
 
+export const getTokenEffects = (
```

Comment:
NP: Since we are not creating some intermediary form from the tokens, I would just forget about `tokenEffects` as a concept, and use `tokens` or `tokenTransfers` everywhere instead,  so the reader knows that he is working with the Transaciton['tokens'] type and not something different. Sorry I believe that I led you to this naming which seems to be useless now as well as the concept of some arbitrary `tokenEffects`